integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/sender/test/SendMailWithSecondaryAccountThroughESBTestCase.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why have you changed the year in license header? License year should when the respective file is created (not the updated year).

Reply:
Fixed with https://github.com/wso2/product-ei/pull/2170